tsconfig.lint.json

```diff
@@ -0,0 +1,16 @@
+/* Avoid clashing types between Cypress and Jest for linting files
```

Comment:
It's always annoying that you're not allowed to use comments in JSON officially...

If the comments are not causing us problems when running programs, I'd like to keep them in, so that we know what we did when we read this code later. 😄

Reply:
It's fine for TS config files since they are based in json5, which allows comments.

Reply:
Let's leave the comments in then! 😄